integration_tests/ipp/container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM zgrab2_service_base:latest
+
+RUN apt-get update
+RUN apt-get install -y cups
+RUN apt-get install -y cups-pdf
+RUN echo "ServerAlias *" >> /etc/cups/cupsd.conf
+RUN echo "FileDevice yes" >> /etc/cups/cupsd.conf
+RUN sed -i 's/Order allow,deny/Order allow,deny\'$'\n  Allow all/g' /etc/cups/cupsd.conf
```

Comment:
Is this missing an escape before the single quote after the $?

Reply:
**Required** ([_pull request commenting with a purpose_](https://gist.github.com/andrewsardone/3fb9746e415d59784e97))

To keep things tidy (fewer disk layers), and following [best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) some of these commands should be doubled up into a single layer/`RUN` command. e.g.
```
RUN apt-get update \
    && apt-get install -y cups cups-pdf \
    …
```

Reply:
Will do. Thanks, flav.

Reply:
@justinbastress I'm not 100% certain, but I don't believe so. It works as is, though looking at it now it definitely seems like it should have one.